edx_event_bus_kafka/consumer/management/commands/consume_events.py

```diff
@@ -0,0 +1,180 @@
+"""
+Management command for listening to license-manager events and logging them
```

Comment:
```suggestion
Management command for listening to specified events and sending them to Django signals.
```
...or something like that.